lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -0,0 +1,689 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""This library facilitates the integration of the prometheus_remote_write interface.
```

Comment:
Charmhub automatically renders an `<h2>` that looks like this: `charms.alertmanager_k8s.v0.alertmanager_dispatch`.

Should probably add a header of our own:
```suggestion
"""# Prometheus remote-write library

This library facilitates the integration of the prometheus_remote_write interface.
```
A single `#` in markdown is rendered as `<h4>` on charmhub.

Reply:
> A single # in markdown is rendered as <h4> on charmhub.

Wait, what? Why? This sounds broken, but I'll go for it for now.